harness/determined/cli/tensorboard.py

```diff
@@ -56,9 +63,9 @@ def start_tensorboard(args: Namespace) -> None:
                             resource_pool=resp["resourcePool"],
                             description=resp["description"],
                             task_type="tensorboard",
+                            currentSlotsExceeded=currentSlotsExceeded,
                         ),
                     )
-
```

Comment:
please avoid extraneous whitespace changes

Reply:
reverted this